#DNA Barcoding
#### Week of : November 23, 2015

### Important terms

* Gel Electrophoresis
* Band migration
* Sanger DNA Sequencing


### Important Concepts
* Gel electrophoresis separates DNA by size
* We use electrophoresis to confirm our PCR worked
* PCR amplified DNA can be sequenced (read) 


## Electrophoresis and sequencing
To help us understand sequencing better we use bioinformativs- interdisciplinary field that develops methods and sfotware toold for understanding bio;ogical data. Using computers to visualize and understand. When sequencing a DNA you need the DNA you wish to sequence,PCR (template), the PCR primer and a polymerase enzyme. We add dideoxynucleotides are added to the reaction. ddNTPs come in different olors because they have two modifications: 1. dideoxynucleotides are missing 2 oxygens and 2. they have a flourescent dye. In the first stage, heating, the DNA strands are heated up, denatures and the double strands are separated. In the annealing stage, the reaction is cooled long enough for the PCR Primer to bond to the single strand DNA. Lastly, there is the Extension stage where the nucleotides float around randomly and if the correct one hits the polymerase, it gets attached. Every one in a while, randomly, instead of a dNTP being incorporated, a ddNTP will be incorporated --> thus causing the DNA strand to terminate (die) because it prevents the matching nucleotide from fitting. Some will died at 8, 100, or even 1 it is all random. We take all the pieces of DNA and put them into a gel called polycrimilide which is much thicker than the first one we used (agorose). We put it in the machine and the pieces are separated by weight/size: the shorter pieces of DNA move quicker thrrough the gel. The polycrimilide gel is special because it gives us high resolution, in the agorose gel you cant tell the difference between 500 and 501 but the polycrimilide gel is so detailed that you can. When our DNA passes throught the gel, a laser  records the color of the ddNTP and you can see it in the electrophorogram. It puts them in size order so you can see what pair it the DNA died at. Because the size corresponds to the position,, we know what the origincal sequence was. We can figure out the original sequence of DNA by size, color and position but you only get to about 1000. The beginning and end sequences aren't reliable

After we have amplified DNA by PCR we need to examine the results. The conventional way to do this is by using [Gel Electrophoresis](https://en.wikipedia.org/wiki/Gel_electrophoresis). 

Review this [animation on DNA electrophoresis](https://www.dnalc.org/resources/animations/gelelectrophoresis.html)

### Questions to answer about electrophoresis

1. What type of gel is used?
2. What is the loading dye and what is the purpose?
3. How does DNA migrate in an electric field?
4. How do we view the stained DNA bands; what stain, what light source is used?


### Your contributions
